webpack.config.shared.js

```diff
@@ -4,6 +4,11 @@ const glob = require('glob');
 
 module.exports = {
   config: {
+    node: {
```

Comment:
What was this change for?

Reply:
That was for the VideoJS Youtube plugin, which isn't included in this PR, I'll remove it.